docs/quickstart.md

```diff
@@ -83,8 +83,9 @@ $ python static_test_script.py mephisto.architect=heroku mephisto.provider.reque
 # Note: my_mturk_user_sandbox is what we used to name the requester
 # when we registered the mturk account in the previous step.
 ```
-The arguments `mephisto.provider.requester_name=my_mturk_user_sandbox` and `mephisto.architect=heroku` will tell the the script to use the mturk sandbox provider and the heroku architect (as opposed to the mock provider and local architect).
+The arguments `mephisto.provider.requester_name=my_mturk_user_sandbox` and `mephisto/architect=heroku` will tell the the script to use the mturk sandbox provider and the heroku architect (as opposed to the mock provider and local architect). Notice that if we're setting a full abstraction (like the architect) we reference it with `mephisto/abstraction=val`, however when we're setting an argument, we use `mephisto.abstraction.argument=val`. This tells hydra whether we're providing an argument value or the name of a configuration to load.
```

Comment:
Hmm - line 81 should now be changed to say `mephisto/architect=heroku`, right?

Reply:
Also, there's still a reference to `mephisto.architect=heroku` in https://github.com/facebookresearch/Mephisto/blob/master/docs/hydra_migration.md

Reply:
Thanks for the flags! Updated all of them now